modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/Billing.java

```diff
@@ -17,7 +18,7 @@
  * Defines the billing model of the managed service.
  */
 @Data
-public class Billing {
+public class Billing implements Serializable {
 
```

Comment:
Add `serialVersionUID` to avoid potential incompatibilities during deserialization.